keps/sig-node/5233-node-readiness-gates/README.md

```diff
@@ -0,0 +1,588 @@
+# KEP-5233: Node Readiness Gates
```

Comment:
If you are hoping to merge this for 1.35, please request a PRR review. You will need to add prod-readiness file and find an approver for that.